marklogic-data-hub-central/ui-custom/src/views/Detail.tsx

```diff
@@ -88,11 +88,11 @@ const Detail: React.FC<Props> = (props) => {
     setExpand(!expand);
   };
 
-  useEffect(() => {
-    if (id !== detailContext?.detail?.uri) {
-      detailContext.handleGetDetail(id);
-    }
-  }, [id])
+  // useEffect(() => {
```

Comment:
@iveyMarklogic, thanks. I've added the following check:
```
!_.isEmpty(userContext.config)) {
```
This avoids an error when accessing the Detail view by pasting a URL into the browser, for example:

http://localhost:3000/detail?recordId=/person/10088.xml

The config.json needs to be loaded as userContext.config, otherwise it will error.

Also, I updated the relationshipsStyle code. This code is included is to shrink the Relationships container in the case where there are no relationships to display. We don't want the empty container taking up excess space.